jemoji.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'jekyll', '>= 3.0'
   s.add_dependency 'html-pipeline', '~> 2.2'
   s.add_dependency 'activesupport', '~> 4.0'
-  s.add_dependency 'gemoji', '~> 2.0'
+  s.add_dependency 'gemoji', '~> 3.0'
 
```

Comment:
It looks like the breaking change is the emoji themselves? What's the motivation for allowing 2.x?